statuses.py

```diff
@@ -58,4 +58,8 @@
    "Not good with words? Offer a hug. Hugs are the ultimate source of comfort.",
    "Be kind. Be lovable. Be huggable.",
    "Hugging is the silent way of saying \"You matter to me\".",
+   "No talk me, just hugs.",
```

Comment:
The grammar here looks a bit off?

Maybe "No talk, just hugs." would be better?

Reply:
Yes, it was intended because of the meme no talk me, im angy haha. But I'll change it :)

Reply:
Ah, my bad, I wasn't familiar with that meme :)